### ISSUE TITLE
Installation of racer fails on fresh macOS install

### ISSUE THREAD

**Issue by vyorkin** [racer-rust/racer#1121]
Steps to reproduce

1. Install macOS
2. 

```
        # install rust
        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
        source ~/.cargo/env
        rustup toolchain add nightly
        rustup default nightly
        rustup component add rust-src
        cargo install bingrep
        cargo install rusty-tags
        rustup component add rustfmt-preview --toolchain nightly
        cargo +nightly install racer
```

error

```
error: `std::intrinsics::transmute` is not yet stable as a const fn
   --> /Users/vyorkin/.cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-rustc_ast-669.0.0/ptr.rs:136:17
    |
136 |                 std::mem::transmute(NonNull::<[T; 0]>::dangling() as NonNull<[T]>)
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: add `#![feature(const_transmute)]` to the crate attributes to enable

error: aborting due to previous error

error: could not compile `rustc-ap-rustc_ast`
```

**Comment by svenstaro** [comment]
Your rust nightly is too old. Try: `rustup default nightly-2020-07-21`

**Comment by svenstaro** [comment]
Fixed by #1122. Check it out.

**Comment by vyorkin** [comment]
Thank you! My suggestion is to update the README.md to mention the `nightly-2020-07-21`, because right now it says that it is enough to just `rustup toolchain add nightly` and `cargo +nightly install racer`
